### PR TITLE
[tx_hash label] [Easy] Add harvest yield tx_hash label 

### PR DESCRIPTION
**Pull request by gentrexha**
Brief comments on the purpose of your changes:

Implement the harvest yield tx hash label

cc @bh2smith for internal review

**For Dune Engine V2**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [x] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### CODE REVIEW

**Comment by gentrexha** [comment]
This will be fixed once #2748 is merged.

File: models/cow_protocol/tx_hash_labels/harvest_yield/ethereum/cow_protocol_tx_hash_labels_harvest_yield_ethereum.sql [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    ('0xc0c293ce456ff0ed870add98a0828dd4d2903dbf'), -- AURA
+                    ('0x6b3595068778dd592e39a122f4f5a5cf09c90fe2'), -- SUSHI
+                    ('0x1f9840a85d5af5bf1d1762f925bdaddc4201f984'), -- UNI
+                    ('0xd533a949740bb3306d119cc777fa900ba034cd52'), -- CRV
+                    ('0x4e3fbd56cd56c3e72c1403e103b45db9da5b9d2b'), -- CVX
+                    ('0x6cacdb97e3fc8136805a9e7c342d866ab77d0957'), -- SWAPR
+                    ('0x48c3399719b582dd63eb5aadf12a40b4c3f52fa2'), -- SWISE
+                    ('0x875773784af8135ea0ef43b5a374aad105c5d39e'), -- IDLE
+                    ('0xd33526068d116ce69f19a9ee46f0bd304f21a51f') -- RPL
+        ) t(harvest_yield_token_address)
```
Comment (bh2smith) [inline models/cow_protocol/tx_hash_labels/harvest_yield/ethereum/cow_protocol_tx_hash_labels_harvest_yield_ethereum.sql:23]: Interesting list. I think there may be many many more of these but you captured some big name brands as well as a few I've never heard of so... 👌

**Review by bh2smith — APPROVED** [review]
I'm seeing a lot of redundancy in these queries. Is there any possible way that this can be made more compact? Or is this the standard setup for labeling?

File: models/cow_protocol/tx_hash_labels/harvest_yield/ethereum/cow_protocol_tx_hash_labels_harvest_yield_ethereum.sql (hunk 1 above)
Comment (gentrexha) [reply models/cow_protocol/tx_hash_labels/harvest_yield/ethereum/cow_protocol_tx_hash_labels_harvest_yield_ethereum.sql:23]: Thanks. The list was suggested from the Karpatkey team.

**Pull request merged**